pkg/confgen/flowlogs2metrics_config.go

```diff
@@ -97,7 +98,84 @@ func (cg *ConfGen) generateFlowlogs2PipelineConfig(fileName string) error {
 			},
 		},
 	}
+	return config
+}
+
+func (cg *ConfGen) GenerateTruncatedConfig(stages []string) map[string]interface{} {
+	parameters := make([]map[string]interface{}, len(stages))
+	for i, stage := range stages {
+		switch stage {
+		case "ingest":
+			parameters[i] = map[string]interface{}{
+				"name": "ingest_collector",
+				"ingest": map[string]interface{}{
+					"type": "collector",
+					"collector": map[string]interface{}{
+						"port":       cg.config.Ingest.Collector.Port,
+						"portLegacy": cg.config.Ingest.Collector.PortLegacy,
+						"hostname":   cg.config.Ingest.Collector.HostName,
+					},
+				},
```

Comment:
I think we now have all these data structures defined in https://github.com/netobserv/flowlogs-pipeline/tree/main/pkg/api
Maybe we should consider reusing them rather than having them repeated here?

Reply:
The high-level of StageParam is defined in a different file:
https://github.com/netobserv/flowlogs-pipeline/blob/467e409460cef7c7f408cc3ab38476a5da1e03aa/pkg/config/config.go#L48-L55

And perhaps the pipeline config builder could become handy
https://github.com/netobserv/flowlogs-pipeline/blob/main/pkg/config/pipeline_builder.go

Reply:
If we want to redo this, let's put it in a separate issue and PR.

Reply:
I created an issue for this
https://github.com/netobserv/flowlogs-pipeline/issues/254